configuration/security.md

```diff
@@ -182,28 +187,36 @@ To disable the firewall, and disable start up on boot, use:
 sudo ufw disable
 ```
 
-Allow a particular port to have access (we have used port 22 in our example):
+What you'll want to do is deny all incoming requests by default, and allow all outgoing requests by default. This will allow you to allow only specific IP addresses to connect in on a specific port, and allow any and all IP addresses to connect out on any and all ports.
+
+To deny all incoming requests by default, use:
+
+```bash
+sudo ufw default deny incoming
+```
+
+To allow all outgoing requests by default, use:
 
 ```bash
-sudo ufw allow 22
+sudo ufw default allow outgoing
 ```
 
-Denying access on a port is also very simple (again, we have used port 22 as an example):
+To disable logging, use:
 
 ```bash
-sudo ufw deny 22
+sudo ufw logging off
```

Comment:
Perhaps this PR would be more useful if it simply added to the existing content, rather than replacing existing content? :confused:

Reply:
Actually I rearranged it, it's still there but it's been changed and it's on a new line (244-246).